pkgs/applications/version-management/git-and-tools/legit/default.nix

```diff
@@ -0,0 +1,35 @@
+{ lib, python3Packages }:
+
+python3Packages.buildPythonApplication rec {
+  pname = "legit";
+  version = "1.1.0";
+  src = python3Packages.fetchPypi {
+    inherit pname version;
+    sha256 = "0kbvczs0rnzazxqx8nc10rp6x9aqgzf318m68nyv1japsq4kig95";
+  };
+
+  propagatedBuildInputs = with python3Packages; [
+    click
+    clint
+    crayons
+    GitPython
+    six
+  ];
+
+  # Unpin dependency versions.
```

Comment:
Are you sure this wouldn't cause breakage? Usually there's a reason upstream pins versions, if you have tested and you think there shouldn't be a reason to pin versions, talk with upstream about it.

Reply:
When i submitted this PR, legit was improperly dumping requirements.txt > install_requires. Removing arbitrary dependency constraints to avoid packaging arbitrary dependency versions seems to be how this is usually handled in nixpkgs. However, this has been corrected in a subsequent release so this can indeed be removed.